horizons/gui/widgets/logbook.py

```diff
@@ -217,7 +217,7 @@ def _icon(image):
 		def _label(text, font='default'):
 			try:
 				return Label(text=unicode(text), wrap_text=True,
-				             min_size=(335, 0), max_size=(335, 508),
+				             min_size=(325, 0), max_size=(325, 400),
```

Comment:
This does not seem to fit with the VBox `max_size` any more after adding the `ScrollArea`.